src/llvm/builder.cr

```diff
@@ -251,6 +251,10 @@ class LLVM::Builder
     Value.new LibLLVM.get_current_debug_location(self)
   end
 
+  def current_debug_location_metadata
+    LibLLVM.get_current_debug_location_metadata(self)
+  end
+
```

Comment:
Like in another comment, this seems to be unused.

Reply:
It used to be used in the code but not anymore, but as I said previously it is quite useful for developers who will use LLVM module.

If you think it may not be useful I will remove it.

Reply:
If we keep debug information then let's put a comment on top with something like "This is unused, but if you ever need to debug X you can do Y". But I'd rather remove debug code.